.eslintrc

```diff
@@ -0,0 +1,31 @@
+root: true
+
+env:
+  node: true
+  browser: true
+  mocha: true
+
+rules:
```

Comment:
Did you list the rules in a specific order here?
If so, could you sort the rules in the same order as on the [doc](http://eslint.org/docs/rules/) please?